src/bp/core/botpress.ts

```diff
@@ -120,11 +120,38 @@ export class Botpress {
     process.exit()
   }
 
+  private async _copyPretrainedAndStopWords() {
```

Comment:
Since your methods don't use any class member, please move the method definitions elsewhere, to avoid bloating botpress.ts